srcpkgs/picom/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'picom'
 pkgname=picom
-version=11.2
+version=12.3
 revision=1
 build_style=meson
 configure_args="-Dwith_docs=true"
-hostmakedepends="pkg-config asciidoc"
+hostmakedepends="cmake pkg-config ruby-asciidoctor"
```

Comment:
Even if that PR had not been a duplicate, there's no part of picom that needs cmake